Project.toml

```diff
@@ -9,29 +9,29 @@ DeepDiffs = "ab62b9b5-e342-54a8-a765-a90f495de1a6"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
-ImageInTerminal = "d8c32880-2388-543b-8c61-d9f865259254"
 LazyModules = "8cdb02fc-e678-4876-92c5-9defec4f444e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+XTermColors = "c8c2cc18-de81-4e68-b407-38a3a0c0491f"
 
 [compat]
 BSON = "0.3"
-Colors = "0.10, 0.11, 0.12"
+Colors = "0.10 - 0.12"
 CSVFiles = "1"
-DataFrames = "0.21, 0.22"
+DataFrames = "0.21 - 0.22"
 DeepDiffs = "1.1"
-Distances = "0.7, 0.8, 0.9, 0.10"
+Distances = "0.7 - 0.10"
 FileIO = "1"
 GR = "= 0.50.1"
 ImageCore = "0.8.1, 0.9"
-ImageInTerminal = "0.3, 0.4, 0.5"
 ImageMagick = "0.7, 1"
 ImageTransformations = "0.8"
 LazyModules = "0.3"
 Plots = "= 1.4.3"
 TestImages = "0.6, 1"
-julia = "1"
+XTermColors = "0.2"
+julia = "1.6"
```

Comment:
@oxinabox @juliohm

is it okay for ReferenceTests to drop 1.0 compatibility?

Reply:
`XTermColors` is [marked](https://github.com/JuliaImages/XTermColors.jl/blob/c988dc276e252764d8a439b3e1996371b29ca795/Project.toml#L14) 1.6, so we have no choice here.

Reply:
Yes, we can drop julia 1.0, if someone needs to use ReferenceTest.jl on an older version of julia they can always use an older version.